### DISCUSSION TITLE
Notes about a repo

### DISCUSSION THREAD

**Discussion by SorinGFS** [community/community#8763] [Other Feature Feedback, Questions, & Ideas]
Often I find an interesting repo and I'm forced to give it a star or to set some watch in order to remember it, but after a while I can't exactly remember why I did that. I think it would be helpful if we had a place to write a note about the repo we are looking at.

**Comment by brianlovin** [comment]
Thanks for the suggestion @SorinGFS — would it be reasonable for the name of the list itself to communicate why a repo was starred? If you have a specific example, that would be helpful for us to understand how you're using the feature 😊

**Comment by SorinGFS** [comment]
I think the proper name for this list would be **Notes** and can be placed right near `Future Ideas`, `Inspiration` and `MyStack` checkboxes located in `Star` component on the upper right corner. 

![Untitled](https://user-images.githubusercontent.com/25417015/148194581-fee3b0b2-ab09-4a4f-89d5-32b1378a1137.png)

Another idea would be to just transform the  `Future Ideas`, `Inspiration` and `MyStack` checkboxes to text fields.

As for the notes we write we should be able to read them in the `Stars` tab near corresponding repository.

**Reply by brianlovin** [reply]
Got it, thanks for the context. We aren't planning to add per-repository level notes, but in the meantime if it's helpful you can add a description to the list itself.

**Reply by SorinGFS** [reply]
> if it's helpful you can add a description to the list itself.

It is not helpful at all, as the title sais I would like to add notes about **a repo**, not about **ALL** repos.
